infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -89,6 +89,12 @@ private void invalidateFirst() {
                 maxConflictWildcard, entry.txVersion, entry.conflictTxStream);
     }
 
+    long firstAddress() {
```

Comment:
can we add an access specifier here?

Reply:
The address is a not a member and could not be a member either.